WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -104,12 +98,6 @@ final class DashboardViewModel: ObservableObject {
         installPendingThemeIfNeeded()
     }
 
-    /// Uploads the answers from the store creation profiler flow
-    ///
-    func uploadProfilerAnswers() async {
-        await storeCreationProfilerUploadAnswersUseCase.uploadAnswers()
```

Comment:
`storeCreationProfilerUploadAnswersUseCase` property can be deleted as well.

Reply:
Removed in d125d80ef40a2ed8f3b2e1eff225b63446d28dd7.